code/controllers/subsystem/jobs.dm

```diff
@@ -390,6 +390,7 @@ var/datum/subsystem/job/SSjob
 		else //We ran out of spare locker spawns!
 			break
 
+
```

Comment:
if you're commiting with something like tortoisegit you should always open the code on the commit window to look for these empty line only instances, you can safely not commit this file without it affecting anything. Of course now that it's been commited it's a little harder to remove in a way that vanishes it from this PR. So don't worry about it for now.